src/frontend/src/components/SearchPage/SearchPage.jsx

```diff
@@ -135,7 +141,10 @@ const SearchPage = () => {
       //  for each with feed guid as key
       if (results.type === 'author') {
         return results.searchResults.map((result) => (
-          <AuthorResult key={result.id} author={result.author} post={result.post} />
+          <div>
```

Comment:
I think the fact that you need to wrap this in a `<div>` means we probably should break this out into its own component.

Reply:
Ahh true. Maybe I can just implement the timeline on the AuthorResult component itself. But regarding the author card, I also wasn't really sure if that was the best way to present it but I was also wanted to know what you thought of it since you mentioned replacing the results with the timeline or 'adding it to the what we have now'. I'll try and think of a better way to show it, but if you have any ideas as well I'm all ears.

Reply:
Maybe we should consider removing the Author card and only showing results?

cc @c3ho, @cindyledev, @agarcia-caicedo, who might have thoughts.

Reply:
For disclosure I am a UI idiot, I am fine iwth removing the author card though.

Reply:
The Author card was removed in #1295, so I recommend rebasing before so you can work on the most recent version of the code.